internal/transport/http2_client_test.go

```diff
@@ -74,10 +73,12 @@ func dialerClientPrefaceLength(_ context.Context, addr string) (net.Conn, error)
 	return &clientPrefaceLengthConn{Conn: conn}, nil
 }
 
-func (hc *framerWriteSettingsConn) Write(b []byte) (n int, err error) {
-	n, err = hc.Conn.Write(b)
+func (fws *framerWriteSettingsConn) Write(b []byte) (n int, err error) {
+	framerValue := 9
+	n, err = fws.Conn.Write(b)
```

Comment:
@hanut19 please do `string(b)` and compare the actual string and throw the error if intended test case string is sent for write.

Please do this for all the 4 test cases

Reply:
done